aries-site/src/pages/components/grid.mdx

```diff
@@ -13,3 +22,201 @@ import { GridExample } from '../../examples';
+
+Specifying the column width as single string will repeat multiple columns of that size, as long as 
+there is room for more. For example, `columns="medium"` will fit as many "medium" columns into the 
+grid as the overall width allows. Within a wide screen, this might allow 3 or 4 columns; within a narrow 
+screen, maybe only 1 or 2.
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/UniformColumns.js"
+  width={{ min: 'medium' }}
+>
+  <UniformColumns />
+</Example>
+
+#### Mixed-Width Columns
+Mixed-width columns allow for control over layouts desiring columns of various widths to appropriately present 
+their contents, establish content hierarchy with wider columns for primary content and narrower for secondary 
+content such as an [activity feed](/templates/dashboards#two-column-dashboard).
+
+Mixed-width columns are specified in an array. The number of items in the array dictate the number of columns 
+in the grid. 
+
+For example, columns specified as `columns={[ 'small', 'xsmall', 'xsmall' ]}` would create a three 
+column grid with corresponding widths: 
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MixedColumns1.js"
+  width={{ min: 'medium' }}
+  height="auto"
+>
+  <MixedColumns1 />
+</Example>
+
+<br /><br />
+
+And, `columns={[ '2/3', '1/3' ]}` would result in two columns with proportional 
+widths:
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MixedColumns2.js"
+  width={{ min: 'medium' }}
+  height="auto"
+>
+  <MixedColumns2 />
+</Example>
+
+
+#### Flexible Columns
+Flexible columns is where Grid really shines. They allow content to elegantly scale and designs to be dynamic.
+
+The main idea is defining columns as a min-max range. Between those bounds, the column is allowed to 
+flex according to its content 
+and the space available to the grid as a whole. This allows content to be consistently displayed in an 
+organized and cohesive fashion across a variety of screen widths.
+
+The min-max range is specified using a two-item array, instead of a single string. 
+The first item is the min and second max.
+
+```
+// 3 columns. First and second columns are defined as min-max 
+// ranges. The third, as a fixed width.
+
+columns={[
+  ['small', 'medium'], // column 1 is able to flex
+  ['medium', 'large'], // column 2 is able to flex
+  'small',             // column 3 is always 'small'
+]};
+```
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MinMaxColumns1.js"
+  width={{ min: 'medium' }}
+  height="auto"
+>
+  <MinMaxColumns1 />
```

Comment:
Wishing there was someway to resize the example inline to understand this behavior better

Reply:
I'll think on that.